package.json

```diff
@@ -36,6 +46,8 @@
     "css-loader": "^0.28.11",
     "grafana-sdk-mocks": "github:grafana/grafana-sdk-mocks",
     "jest": "^23.0.1",
+    "prettier": "1.10.2",
+    "pretty-quick": "^1.4.1",
```

Comment:
Could we use `lint-staged` instead? That's what we use in Grafana.

Reply:
changed to husky+lint-staged